### DISCUSSION TITLE
Causes high server costs

### DISCUSSION THREAD

**Discussion by yukipioa** [mozilla/hubs#5303] [Hubs Cloud Questions]
Hello.
I have a question.

Could you tell me what is causing the higher server costs?
Example: Does the placement of a lot of video or 3D increase data traffic and server costs?
Or, does the number of simultaneously connected users increase the cost?
Does it become more expensive if the time spent on the site is longer?
Does the cost increase with voice conversations?

Many Thanks!

**Comment by yengalvez** [comment]
I saw that the biggest problem is with the database. Hubs cloud use a fixed database with high cost. Someone created a tutorial to change it to a lowcost database (I don't understan very well about it). I hope Hubs Cloud developers can fix it with this options, because actually the cost is massive if you want to have inmediate access to the server without pause the database.

**Reply by yukipioa** [reply]
@gitvirtualmente 
Thank you for replying me!
I understand that the main problem is that Hubs Cloud uses a costly database.
Do you know the best way to reduce costs other than changing the database?
Would it be to eliminate the voice conversation feature? Or is it to place less 3DCG, etc.?

Also, does the cost vary depending on whether I set up YouTube or embed the video?
If so, which would be more cost effective?

**Reply by yengalvez** [reply]
Somebody created a mod to use a different database, but i don't know hot it works haha
https://gist.github.com/wmurphyrd/89c66af8a3a193c2710e1ee9443d6aca
